tests/ga/extension_emulator.py

```diff
@@ -145,25 +144,26 @@ def __init__(self):
     def add(self, ext_name, ext_ver, ext_cmd):
         self._queue.append((ext_name, ext_ver, ext_cmd))
 
-    def compare(self, *cmds):
+    def compare(self, *expected_cmds):
         """
         Verifies that any and all recorded invocations appear in the provided command list in that exact ordering.
 
-        Each cmd in cmds should be a tuple of the form (ExtensionEmulator, ExtensionCommandNames).
+        Each cmd in expected_cmds should be a tuple of the form (ExtensionEmulator, ExtensionCommandNames).
```

Comment:
NIT: Rename to - 
`Each cmd in expected_cmds should be a tuple of the form (ExtensionEmulator, ExtensionCommandNames object).`

Reply:
Is this supposed to say `...ExtensionCommandName object...`?

((Note the lack of an s))

Reply:
No like ExtensionCommandNames  (as in the name of the class) and object. Essentially `ExtensionCommandNames object` would mean an object of the ExtensionCommandNames class. Does that make sense? I'm confused too now lol